### DISCUSSION TITLE
Automatically close figure

### DISCUSSION THREAD

**Discussion by lkries** [highfestiva/finplot#313] [Q&A] [answered]
It's possible to automatically close figure after some time showing it?

Example:

fplt.show()
fplt.pause(3)
fplt.close()

Thanks

**Comment by highfestiva** [comment, marked as answer]
You might be able to do something like this:
```python
fplt.timer_callback(fplt.close, 3)
fplt.show()
```
The reason for this roundabout way is how the PyQt event loop works.

**Reply by lkries** [reply]
Thank you!
